### DISCUSSION TITLE
Q about using Artisan with a Behmor

### DISCUSSION THREAD

**Discussion by ecelis813** [artisan-roaster-scope/artisan#919] [Q&A]
Please excuse me if this question is stupid or if it has been answered 1000 times. I am a home roaster using a Behmor 1600AB roaster and wish to use Artisan using a Mac, just to manually log my roasts. I don't have any temperature probes in the roaster that could be used to the Artisan. First question is after clicking "START" the time initiates but I don't know how to log in the internal chamber temperature (obtained by clicking B in the Behmor), which I wish to do every minute or so. The only way that I can get points is by moving the "Heater" slide bar on the left of the screen, which goes from 0 to 100 (I guess this is the % of heater power). Is there a way to change the scale to temp (F)?  Also I would like to log in the time points where I modify the heating power during the roast, which in the Behmor goes from P5 (100%) to P4 (75%), P3 (50%), P2 (25%) and P0 (0%). My second question is that I get this message on the top left of the screen "Serial Exception: Unable to open serial port". Is there a way to correct this or do I simply ignore it?

Thanks for any help

**Comment by MAKOMO** [comment]
So you want to use Artisan in manual mode (without any hardware to input temperature data). Do a Factory Reset (under menu Help) to bring your configuration back to the initial settings to get rid of this Serial Exception that comes from some misconfiguration. Then open the Events dialog (menu Config >> Events) and tick "Event Button" in its first tab. This enables the yellow events button.

To input temperature data at any point in time during recording, just press that yellow Event button and put the temperatures in the popup dialog. 

You can input your heater % using the Burner slider. Activate that slider by opening again the Events dialog (menu Config >> Events) on its third tab "Sliders" and tick the flag of the Burner slider entry (last line).

Now you should see that slider being displayed after START on the left side. Just move it to enter a corresponding event.

**Comment by ecelis813** [comment]
Thanks for the help. I'm stuck after opening the "Buttons" window in Configurations > Events. Do I create one by clicking "Add"? If so? What info I need to make this a temp input?



Dr. Esteban Celis
Minneapolis, MN

**Comment by ecelis813** [comment]
Forget my last email. I think it worked! I guess BT is beans temp and ET external temp?

Dr. Esteban Celis
Minneapolis, MN

**Comment by MAKOMO** [comment]
Oh, of course you can define event buttons corresponding those Px buttons on the Behmor (what about P1?) and use those to input heater changes instead of the slider. Yes, use "Add" to add an event button in the "Buttons" tab of the Events dialog to end up with something like this:

BT: bean temperature
ET: environmental temperature (the temperature of the air just above the beans)

You might also choose "Combo" for the Markers configuration in the Config tab of the Events dialog and put a unit to the Burner slider (last column) under the Slider tab of the Events dialog to have the heater percentage rendered directly with the event marker as shown below.

<img width="1010" alt="Screenshot 2022-07-17 at 07 41 31" src="https://user-images.githubusercontent.com/13909011/179385615-ab278e83-15c2-45fb-8abc-6d544320a14d.png">
<img width="1010" alt="Screenshot 2022-07-17 at 07 41 41" src="https://user-images.githubusercontent.com/13909011/179385617-4fe1f26a-9065-4efa-9153-91cd9c0dc875.png">
<img width="584" alt="Screenshot 2022-07-17 at 07 41 55" src="https://user-images.githubusercontent.com/13909011/179385619-875a2b75-d2a6-42d4-87ab-2c854fa090d9.png">
